src/control/lib/control/pool.go

```diff
@@ -344,7 +343,8 @@ type (
 	// PoolQueryReq contains the parameters for a pool query request.
 	PoolQueryReq struct {
 		poolRequest
-		ID string
+		ID      string
+		InfoBit int32
```

Comment:
I declared it as an Int32 because the equivalent C constant are defined as enum. However, for the GO part, using an uint32 for a bit mask is more suitable => I will change it .

- [ ] Convert int32 to uint32